src/main/java/com/cosium/vet/git/RemoteUrl.java

```diff
@@ -2,29 +2,19 @@
 
 import com.cosium.vet.utils.NonBlankString;
 
-import java.net.MalformedURLException;
-import java.net.URL;
-
 /**
  * Created on 21/02/18.
  *
  * @author Reda.Housni-Alaoui
  */
 public class RemoteUrl extends NonBlankString {
 
-  private RemoteUrl(String value) {
-    super(value);
-  }
-
-  public static RemoteUrl of(String url) {
-    return new RemoteUrl(url);
-  }
+    private RemoteUrl(String value) {
```

Comment:
Please use Google Java Format

Reply:
Also could you add 2 tests:
- one building an http url
- another one building an ssh url